src/components/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Heading } from '@chakra-ui/react';
+import Navbar from "./Navbar";
+
+const Header = () => {
+  return (
+    <header className="App-header">
+        <Navbar />
+        <Heading as="h1" size="lg" fontWeight="bold">
+          Magic Kitchen
```

Comment:
Ask Anna if she wants the full logo here or if she wants the page name?

Reply:
@ah0606